offscreen-canvas/convert-to-blob/offscreencanvas.convert.to.blob.html

```diff
@@ -144,5 +144,22 @@
     }));
 }, "Test that call convertToBlob on a OffscreenCanvas with size 0 throws exception");
 
+async_test(function(t) {
+    var img = new Image();
+    img.src = "/images/green.png";
+    img.crossOrigin = "anonymous";
+    img.onload = t.step_func_done(() => {
+        var offscreenCanvas = new OffscreenCanvas(10, 10);
+        var ctx = offscreenCanvas.getContext("2d");
+        ctx.drawImage(img, 0, 0);
+        offscreenCanvas.convertToBlob().then(t.step_func_done(function() {
+            assert_false("convertToBlob didn't throw, but should");
+        }), t.step_func_done(function(e) {
+            assert_true(e instanceof DOMException);
+            assert_equals(e.name, "SecurityError");
+        }));
```

Comment:
This would be a good place to use `promise_test` + `promise_rejects`, but I see this is consistent with the other subtests in this file, so I won't block on it.